Makefile

```diff
@@ -68,9 +74,30 @@ publish-version:
 	  exit 1; \
 	fi
 
-tag:
-	$(GIT_TAG) --message v$(VERSION) v$(VERSION)
-	$(GIT_PUSH) origin refs/tags/v$(VERSION)
+publish-pip:
+	python setup.py sdist bdist_wheel
+	pip install twine
+	twine upload dist/*
+
+publish-dockerhub:
+	docker tag hokusai:latest artsy/hokusai:$(VERSION)
+	docker push artsy/hokusai:$(VERSION)
+	docker tag hokusai:latest artsy/hokusai:latest
+	docker push artsy/hokusai:latest
+
+publish-github:
+	$(AWS) s3 cp \
```

Comment:
This AWS cp copy block is used a couple of other times in this Makefile. Could it make sense / is it possible to extract another "Make function" and use that instead?

Reply:
whoops I misread the order of the args here! carry on